Include/GWCA/Context/WorldContext.h

```diff
@@ -76,7 +76,7 @@ namespace GW {
         /* +h062C */ Array<void *> h062C;
         /* +h063C */ uint8_t h063C[68];
         /* +h0680 */ Array<void *> h0680; // Struct size = 0x138 i think, has a vtable
-        /* +h0690 */ uint32_t salvage_session_id;
+        /* +h0690 */ uint32_t district_number;
```

Comment:
Hum, that's odd. Are you 100% on that change?

Reply:
Yeh I intercepted the `InstanceLoadInfo` StoC packet and overrode the district number with something like 112, then checked which variable it was updating.

Reply:
Ah no this isn't right, will take another look and update with another commit